grammar.js

```diff
@@ -178,6 +181,46 @@ module.exports = grammar({
         'end',
       ),
 
+    if: $ => seq(
+      'if',
+      field('condition', $._expression),
+      choice($._terminator, field('consequence', $.then)),
+      field('alternative', optional(choice($.else, $.elsif))),
```

Comment:
This currently allows:
```crystal
if true
  puts 1
else
  puts 2
elsif false
  puts 3
end
```

Also, the stdlib AST has a structure similar to:

```
(if
  (condition)
  (body)
  (else
    (if
      (condition)
      (body)
      (else
        (body)))))
```

Basically nesting the elsifs as ifs inside the else branch. This also allows for multiple elsif.